libs/utils/utils.js

```diff
@@ -689,6 +690,31 @@ export function convertStageLinks({ anchors, config, hostname, href }) {
   });
 }
 
+async function getImsCountry() {
```

Comment:
I don't think `utils` is the best place for this method, as it would be loaded on every single project and every single visitor, regardless of their logged-in status. I would rather load this conditionally.

Reply:
I moved all my changes to autoblock `blocks/m7business/m7business.js`